shared/constants/teams.js

```diff
@@ -42,11 +41,21 @@ export const makeRequestInfo: I.RecordFactory<Types._RequestInfo> = I.Record({
   username: '',
 })
 
-export const teamRoleByEnum = invert(RPCTypes.teamsTeamRole)
+export const teamRoleByEnum = ((m: {[Types.MaybeTeamRoleType]: RPCTypes.TeamRole}) => {
```

Comment:
Is there a benefit to this over `invert`?

Reply:
`invert` (and lodash in general, it seems) doesn't play well with flow, which is why there were no type errors. I suppose I could force `teamRoleByEnum` to have the right type, but I feel like the $ForceType below will get fixed before `invert` will type-check properly.

Reply:
was that before or after lodash-es. it should be typed

Reply:
This is after. It types `invert` as any in the following:

```
import {invert} from 'lodash-es'

const teamRoleByEnum2 = invert(RPCTypes.teamsTeamRole)
```

Reply:
Any lodash function that I've seen, for that matter